front/app/containers/IdeasShow/components/RightColumnDesktop/index.tsx

```diff
@@ -82,46 +89,48 @@ const RightColumnDesktop = ({
       className={className}
     >
       <Box display="flex" flexDirection="column">
-        <Box
-          padding="20px"
-          borderRadius="3px"
-          background={colors.background}
-          mb="12px"
-        >
-          {(ideaIsInParticipationContext || commentingEnabled) && (
-            <>
-              {showReactionControl && (
-                <Box pb="23px" mb="23px">
-                  <ReactionControl
-                    styleType="shadow"
-                    ideaId={ideaId}
-                    size="4"
-                  />
-                </Box>
-              )}
-              <Box pb="23px" mb="23px" borderBottom="solid 1px #ccc">
-                {phase &&
-                  ideaIsInParticipationContext &&
-                  votingConfig?.getIdeaPageVoteInput({
-                    ideaId,
-                    phase,
-                    compact: false,
-                  })}
-              </Box>
-              {commentingEnabled && (
-                <Box mb="10px">
-                  <GoToCommentsButton />
+        {showInteractionsContainer && (
```

Comment:
Only think I changed here was I wrapped the grey interactions box with this "showInteractionsContainer" variable).